syntax = "proto3";

package mprpc;
message RpcHeader
{
    bytes service_name = 1;
    bytes methond_name = 2;
    uint32 args_size = 3;
}